Hardware/Hardware/CardReader/StatementDescriptor.swift

```diff
@@ -0,0 +1,40 @@
+/// A property wrapper to return a string where:
+/// - The characters <>"' are replaced.
+/// - The maximum length of the string is 22 characters
+/// We do this to add an extra layer of validation to the values that we
+/// pass to the Stripe Terminal SDK when creating a payment intent:
+/// https://stripe.dev/stripe-terminal-ios/docs/Classes/SCPPaymentIntentParameters.html#/c:objc(cs)SCPPaymentIntentParameters(py)statementDescriptor
+/// If we pass strings longer than 22 characters or that contain any of the
+/// characters that are not allowed, the Stripe Terminal SDK will crash.
+@propertyWrapper
```

Comment:
On the web side, we found it useful to trim leading (or trailing) spaces too - for some reason people would enter those and then the descriptor string would get truncated in unexpected ways